docs/src/main/asciidoc/opentracing.adoc

```diff
@@ -196,6 +196,38 @@ quarkus.datasource.jdbc.driver=io.opentracing.contrib.jdbc.TracingDriver
 quarkus.hibernate-orm.dialect=org.hibernate.dialect.PostgreSQLDialect
 ----
 
+
+=== Kafka
+
+The https://github.com/opentracing-contrib/java-kafka-client[Kakfa instrumentation] will add a span for each message sent to or received from a Kafka topic, to enable it, add the following dependency to your pom.xml:
+
+[source, xml]
+----
+<dependency>
+    <groupId>io.opentracing.contrib</groupId>
+    <artifactId>opentracing-kafka-client</artifactId>
+    <version>${opentracing-kafka.version}</version>
+</dependency>
+----
+
+It contains OpenTracing interceptors that must be registered on Kafka producers and consumers.
+
+If you followed the link:kafka[Kafka guide], the interceptors can be added on the `generated-price` and the `prices` channels as follows:
```

Comment:
Maybe mentioned that interceptors are comma-separated (if I'm not mistaken)

Reply:
I added a note

Reply:
Thanks!